src/main/java/com/miquido/validoctor/multirule/MultiRule.java

```diff
@@ -61,7 +62,7 @@
   public static <T> MultiRule<T> of(ReducerRule<T, ?>... reducerRules) {
     int sumSizes = Stream.of(reducerRules)
         .map(rule -> rule.getProperties().size())
-        .reduce((sumSize, size) -> sumSize + size).orElse(0);
+        .reduce(Integer::sum).orElse(0);
```

Comment:
Just FYI, you can replace it by 
` int sumSizes = Stream.of(reducerRules)
        .mapToInt(rule -> rule.getProperties().size())
        .sum();`